libV2/tasks/Reindex.js

```diff
@@ -69,6 +69,8 @@ class ReindexTask extends BaseTask {
                     labels: {
                         [level]: resource,
                     },
+                    // eslint-disable-next-line camelcase
+                    no_reindex: false,
```

Comment:
is `false` the default value? Wondering since in this PR I don't see any place where it would change the current behavior, since I expect `<% $no_reindex NOT %>` to always be true then. I may totally be wrong, just wondering.

Also it may (or not) make more sense to reverse the logic with a `reindex` flag (or named differently) to avoid double negations.

Reply:
this should be `true`, leftover from testing. The reason for the inverted logic: warpscript returns a `NULL` value for keys that aren't present like javascript, which allows us to introduce this flag without upstream callers being aware as it needs to be explicitly set to `true`.

Reply:
OK makes sense, the `false` value was confusing me :)